### ISSUE TITLE
Add command line option for remoteWrite timeout for vmalert

### ISSUE THREAD

**Issue by stevenbeaullieu** [VictoriaMetrics/VictoriaMetrics#3408]
**Is your feature request related to a problem? Please describe.**
Digging through the code, it looks like there is a 30s timeout on remoteWrite requests in vmalert with no way to configure it.

**Describe the solution you'd like**
I would like a command line option to set a remoteWrite timeout on vmalert.  It looks like vmagent has a flag `remoteWrite.sendTimeout`, although it's an array and I'm not sure how that works.  I would think there'd be a single timeout, otherwise which would it choose?

**Describe alternatives you've considered**
Alternative is to accept the default 30s timeout.  I actually haven't seen this happen too often, but from what I could see is it will attempt the send 5 times, so that would be 2.5 minutes if it timed out every time.  When I saw it in our logs it only timed out once and worked the second time.  Since the payload contains the timestamps it probably doesn't even matter when the write happens, so this isn't a huge priority.

**Additional context**
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/v1.79.5-cluster/app/vmalert/remotewrite/remotewrite.go#L74

**Comment by zekker6** [comment]
> I would like a command line option to set a remoteWrite timeout on vmalert. It looks like vmagent has a flag remoteWrite.sendTimeout, although it's an array and I'm not sure how that works. I would think there'd be a single timeout, otherwise which would it choose?

This flag accepts an array of values in order to allow overriding this setting on per-endpoint basics. This will allow to use the following:
```
-remoteWrite.url=http://vm:8428/api/v1/write -remoteWrite.url=http://vm2:8428/api/v1/write -remoteWrite.sendTimeout=30s -remoteWrite.sendTimeout=300s
``` 
Which will configure 30s timeout for `http://vm/...` and 300s for `http://vm2/...`

**Comment by valyala** [comment]
The `-remoteWrite.sendTimeout` command-line option is available in `vmalert` starting from [v1.85.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing this feature request as done.
